source/slang/hlsl.meta.slang

```diff
@@ -6028,9 +6036,7 @@ T exp2(T x)
     __target_switch
     {
     case glsl:
-        if (__isHalf<T>())
-            __intrinsic_asm "exp2($0)";
-        __intrinsic_asm "exp2(float($0))";
+        __intrinsic_asm "exp2($0)";
```

Comment:
I think this was because exp2 isn't defined for double.

Reply:
Good catch.
It was giving me some trouble and I had to remove "float".
I may have to come back to this later.